content/Introduction/architecture.mdx

```diff
@@ -73,4 +73,120 @@ Security is foundational to Agentuity's design:
 
 - **Container Isolation** - Each agent operates in its own isolated environment
 - **Encrypted Communications** - All agent-to-agent communication is encrypted
-- **Secure Deployment** - Protected deployment pipeline from development to production
\ No newline at end of file
+- **Secure Deployment** - Protected deployment pipeline from development to production
+
+## Project Conventions
+
+Agentuity projects follow specific conventions that enable seamless development, testing, and deployment. Understanding these conventions helps you work with existing projects or create new ones from scratch.
```

Comment:
Something like:
"Agentuity projects follow specific conventions in order to take advantage of the deployment and cloud platform Agentuity offers.  While we don't dictate which framework to use, we do have a few things you have to follow in your file system.  Understanding these conventions are important."